docs/detections/rules-ui-manage.asciidoc

```diff
@@ -2,11 +2,21 @@
 [role="xpack"]
 == Manage detection rules
 
+The Rules page allows you to view and manage all prebuilt and custom detection rules.
+
+[role="screenshot"]
+image::images/all-rules.png[The Rules page]
+
+You can sort the rules by clicking on the *Rule*, *Last updated*, or *Enabled* column header. 
+
+TIP: To sort by any other column, enable the *Technical preview* toggle above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```

Comment:
A UX writing thing - we are trying to steer away from "enable/disable the toggle". 

```suggestion
TIP: To sort by any other column, switch the *Technical preview* toggle on above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```